apps/frontend/src/utils/user.ts

```diff
@@ -5,10 +5,17 @@ export const getFullUserName = (
 ): string => (user ? `${user.firstname} ${user.lastname}` : 'None');
```

Comment:
I was just wondering if we need to do the same `preferredname` logic here as well. I see that this function `getFullUserName` is quite similar to the `getFullUserNameWithEmail` but it doesn't use the preferred name logic. Maybe I am wrong but just wanted to make it clear.